pages/ui-components/overview.md

```diff
@@ -8,7 +8,7 @@ type: docs
 redirect_from:
   - /components/
   - /form-controls/
-in_page_nav: false
+in_page_nav: "h2"
```

Comment:
I don't think we need a side nav on this page. The invitation to "Find a USWDS Component" already appears "above the fold," so IMO the sidenav is redundant and looks little awkward.

Reply:
Removed in [cd074b6](https://github.com/uswds/uswds-site/pull/2595/commits/cd074b67d0a0f0cc64d8ac42066ca8b06c50793e)